wp-content/themes/core/components/routes/not_found/css/not_found.pcss

```diff
@@ -0,0 +1,30 @@
+.not-found {
```

Comment:
This is great. One small request: can you add comment blocks in this partial. Take a look at [the Accordion component](https://github.com/moderntribe/square-one/blob/f7f5e6114cff5791d7cb6c0756c0ad31d061ab39/wp-content/themes/core/components/accordion/css/accordion.pcss) (or others) for an example of a comment block at the top of the file and before each selector.